client/src/components/add-job-post-component/job-info-form/index.jsx

```diff
@@ -0,0 +1,252 @@
+  OutlinedInput,
+  InputAdornment,
+  FormHelperText,
+  Autocomplete,
+} from "@mui/material";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider, DatePicker } from "@mui/x-date-pickers";
+import dayjs from "dayjs";
+import { JobLeadContainer, H3 } from "../index.styles";
+import { JOB_TYPES, COMPENSATION_RATES } from "../../../utils/constants";
+
+function AddJobDetails({ jobPostData, setJobPostData }) {
+  const handleInputChange = (input, field) => {
+    setJobPostData({ ...jobPostData, [field]: input });
+  };
+  return (
+    <JobLeadContainer>
+      <H3 style={{ paddingLeft: "2%" }}>Job Information</H3>
+
+      <Container
+        disableGutters
+        maxWidth={false}
+        sx={{
+          display: "flex",
+          paddingLeft: 0,
+          alignItems: "top",
+          justifyContent: "center",
+          flexFlow: "wrap",
+        }}
+      >
+        {/* Job Title Field */}
+        <TextField
+          fullWidth
+          sx={{ m: 1, width: "96%" }}
+          id="title"
+          label="Job Title"
+          value={jobPostData.title}
+          onChange={(e) => handleInputChange(e.target.value, "title")}
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Employer Name Field */}
+        <TextField
+          fullWidth
+          sx={{ m: 1, width: "96%" }}
+          id="employer"
+          label="Employer Name"
+          value={jobPostData.employer}
+          onChange={(e) => handleInputChange(e.target.value, "employer")}
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Location Field */}
+        <TextField
+          fullWidth
+          sx={{ m: 1, width: "96%" }}
+          id="location"
+          label="Location"
+          value={jobPostData.location}
+          onChange={(e) => handleInputChange(e.target.value, "location")}
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Minimum Compensation Field */}
+        <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+          <InputLabel id="minCompensationLabel">
+            Minimum Compensation
+          </InputLabel>
+          <OutlinedInput
+            id="minCompensationLabel"
+            type="number"
+            startAdornment={<InputAdornment position="start">$</InputAdornment>}
+            label="Minimum Compensation"
+            inputProps={{ min: 0 }}
+            value={jobPostData.rate_of_pay_min}
+            onChange={(e) => {
+              const { value } = e.target;
+              if (/^\d*\.?\d*$/.test(value))
+                handleInputChange(value, "rate_of_pay_min");
+            }}
+            required
+            InputLabelProps={{ required: false }}
+          />
+        </FormControl>
+
+        {/* Maximum Compensation Field */}
```

Comment:
Maximum Compensation Field can be less than the Minimum Compensation Field. I'm not sure if we need to account for this, but I wanted to note it just in case!

Reply:
Good catch! Updated the code to reflect these comments 
<img width="863" alt="image" src="https://github.com/user-attachments/assets/846a8ddd-ef2e-4496-a21d-242f4e3d4449" />
<img width="820" alt="image" src="https://github.com/user-attachments/assets/acf8c84f-a8e8-4676-9dd1-beaff129d5db" />